### PR TITLE
OSDOCS-9766 z-stream RNs OCP 4.14.14

### PR DESCRIPTION
**Pull request by cbippley**
Version(s): 4.14

Issue:[ [OSDOCS-9766](https://issues.redhat.com//browse/OSDOCS-9766)](https://issues.redhat.com/browse/OSDOCS-9766)

Link to [docs preview:](https://72067--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-14)

QE review:
Not needed

Additional information: errata links will not work until merge date (feb 28)

### CODE REVIEW

**Review by bburt-rh — APPROVED** [review]
Looks good overall! I noted a couple of ISG issues to clear up plus a few suggestions to help tighten things up a bit.

I assume that the links to the RHSAs are broken in the rendered preview and won't work until the release is live, right?

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -3279,6 +3279,42 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+[id="ocp-4-14-14-wherabouts-cron-schedule"]
+===== Adding "eu-es" region support for IPI
+
+* Previously, the installation program failed to install a cluster on IBM Cloud VPC on the "eu-es" region, although it is supported. With this update, the installation program successfully installs a cluster on IBM Cloud VPC on the "eu-es" region. (link:https://issues.redhat.com/browse/OCPBUGS-19398[*OCPBUGS-19398*])
+
+[id="ocp-4-14-14-IC-updates-troubleshooting"]
+===== Improved troubleshooting for IC updates
+
+* When updating from {product-title} 4.13 to {product-title} 4.14.14 and beyond, if a cluster is not making progress during phase 1 or phase 2 of the update to OVN interconnect configmap (IC) a cluster admin can now edit the IC and add a new key, `fast-forward-to-multizone`, to bypass the two-phase update and apply the final 4.14 YAMLs for ovn-kubernetes. (link:https://issues.redhat.com/browse/OCPBUGS-26573[*OCPBUGS-26573*])
```
[the remark this answers is not in the document]
Comment (bburt-rh) [reply release_notes/ocp-4-14-release-notes.adoc:3311]: Or maybe:

`* Previously, during an update from {product-title} 4.13 to {product-title} 4.14.14 or later, a cluster could become stuck while updating to OVN interconnect configmap (IC) during phase 1 or phase 2. With this release, a cluster admin can now edit the IC and add the `fast-forward-to-multizone` key, which bypasses the two-phase update and applies the final 4.14 YAML files for ovn-kubernetes. (link:https://issues.redhat.com/browse/OCPBUGS-26573[*OCPBUGS-26573*])`

**Review by kelbrown20 — COMMENTED** [review]
LGTM!

**Pull request merged**